gamify-impl/src/main/resources/api-spec.yaml

```diff
@@ -102,3 +162,17 @@ components:
         token:
           type: string
           nullable: false
+
+    # Category
+    Category:
+      type: object
+      properties:
+        name:
+          type: string
+          nullable: false
+        title:
+          type: string
+          nullable: false
+        description:
+          type: string
+          nullable: false
```

Comment:
This is not the same ending as the other `api-spec.yaml` file, so the CI will not succeed.